src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs

```diff
@@ -264,13 +260,28 @@ internal IEnumerable<string> CreateArgument()
                 {
                     allArgs.Add("--testAdapterPath:" + ArgumentEscaper.HandleEscapeSequenceInArgForProcessStart(this.VSTestTraceDataCollectorDirectoryPath));
                 }
+                else
+                {
+                    if (isCollectCodeCoverageEnabled)
+                    {
+                        // Not showing message in runsettings scenario, because we are not sure that code coverage is enabled.
+                        // User might be using older Microsoft.NET.Test.Sdk which don't have CodeCoverage infra.
+                        Console.WriteLine(Resources.UpdateTestSdkForCollectingCodeCoverage);
```

Comment:
Nit: String format

Reply:
What is the solution for user with runsettings then?

Reply:
string format not required here. Not keeping the runsettings knowledge in msbuild task as mentioned in above comment. There is no perf impact because the path is not getting added to resolution directory. 

Users who use runsettings gets general message "Data collection : Could not find data collector 'Code Coverage'", we will create doc for this scenario.